apps/main/src/store/createQuickSwapSlice.ts

```diff
@@ -35,25 +35,52 @@ type SliceState = {
 
 const sliceKey = 'quickSwap'
 
-// prettier-ignore
 export type QuickSwapSlice = {
```

Comment:
`// prettier-ignore` I would prefer if this is not deleted, it is easier to read when the types are all in one line.

Reply:
Maybe we should format the definition differently to make it easier to read? 🤔
Disabling the linters is not good

Reply:
okay.